prqlc/bindings/prqlc-c/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "prqlc-c"
 publish = false
+rust-version = "1.73.0"
```

Comment:
I am not sure if this is the ideal situation as I believe the MSRV is modified by the cli feature. (All downstream crates that don't use the cli feature would also be affected by this)